app/controllers/other_duties_controller.rb

```diff
@@ -21,7 +21,7 @@ def edit
 
   def index
     @volunteer_duties = if current_user.casa_admin?
-      generate_other_duty_list(Volunteer.all)
+      generate_other_duty_list(Volunteer.where(casa_org_id: current_user.casa_org_id))
```

Comment:
the preferred way to do this is to use a `policy_scope` from https://github.com/varvet/pundit

Reply:
Sure, I will look into this.